CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## 2020-10-30
+
+- Added a package `@excalidraw/utils` to provide utilities functions
```

Comment:
It's better to have changelog for `packages/utils` inside `packages/utils` because
1. The versions would be independently published so it's hard to find the log in the global changelog
2. If every package will have its own changelog it would be easier to scan through it.

If we had all packages version incremented when any change is made the global changelog would be better.

Reply:
Yes that a good idea, I will create a dedicated changelog :+1:

Reply:
Done: https://github.com/excalidraw/excalidraw/pull/2246/files#diff-9293e136acee2fa3acc146cbdc15b39c573e42bc4e3b14acb20aa461950265f8